src/gluonts/model/forecast.py

```diff
@@ -637,10 +637,10 @@ def __repr__(self):
         )
 
     def to_quantile_forecast(
-        self, quantiles: List[Union[float, str]]
+        self, quantiles: List[str]
     ) -> "QuantileForecast":
         return QuantileForecast(
-            forecast_arrays=np.array([self.quantile(q) for q in quantiles]),
+            forecast_arrays=np.array([self.quantile(q) if q != "mean" else self.mean() for q in quantiles]),
```

Comment:
This line is probably too long and `black` is complaining. You can run `black src` and commit the changes it applies, to fix it.